screens/NotificationScreen.tsx

```diff
@@ -30,7 +38,7 @@ async function sendPushNotification(expoPushToken: string) {
     },
     body: JSON.stringify(message),
   });
-}
+}*/
 
```

Comment:
brauchen wir den auskommentierten Code noch oder kann der raus?

Reply:
Der Code wird in Zukunft wahrscheinlich nochmal verwendet, deswegen lasse ich ihn gerade mal drin. Falls er in Zukunft nicht gebraucht wird, wird er entfernt